docs/fomega/cek-cps-experiments/results/cps-times-Fac

```diff
@@ -0,0 +1,22 @@
+n usr sys mem
```

Comment:
Did you mean to check in the results?

Reply:
If you did, maybe we can generate the graphs during the build instead of checking them in?

Reply:
I just thought I'd save the results since there's more information in there than appears in the graphs.  The graphs are generated using R (see RUNNING.md), and putting that into nix might be a bit heavyweight just to draw some graphs.This isn't a terribly important document anyway, so I don't know if it's worth complicating the build.